tests/unit/steps/UnzipStepRunnerTest.php

```diff
@@ -73,9 +73,9 @@ public function testUnzipFileWhenUsingAbsolutePath() {
 		$extracted_file_path = $this->runtime->resolvePath( 'dir/test_zip.txt' );
 		$step->setExtractToPath( Path::getDirectory( $extracted_file_path ) );
 
+		$this->expectException(\Exception::class);
```

Comment:
Note:
Instead of expecting these exceptions which note an unimplemented, #123 marks each of the tests ask explicitly skipped because of the missing implementation and adds a TODOs to stop skipping once the step is implemented